apps/src/aichat/views/ChatMessage.tsx

```diff
@@ -91,13 +94,42 @@ const displayAssistantMessage = (status: string, chatMessageText: string) => {
   }
 };
 
+const displayModelUpdateMessage = (
```

Comment:
minor: thoughts on making this a separate component (maybe still lives in this file) as opposed to a render function?

Reply:
I think for consistency with other very similar render functions (displayAssistantMessage, etc) I'll leave here for now, but definitely up to refactor later if preferred.